### ISSUE TITLE
client libs signing subject error

### ISSUE THREAD

**Issue by nconrad** [PATRIC3/Workspace#36]
When I trying to use the python client with a username and password, I was getting the following from the workspace service:

api.patric.workspace.ServerError: JSONRPCError: -32603. Token validation failed: Failed to verify token: Token signed by unrecognized source: https://nexus.api.globusonline.org/goauth/keys/fb4f1e06-322c-11e5-9bf4-123139260d4e at /disks/p3/deployment/lib/Bio/KBase/AuthToken.pm line 481.

It looks like the bad method is https://github.com/PATRIC3/Workspace/blob/master/lib/biop3/Workspace/WorkspaceClient.py#L27

**Comment by olsonanl** [comment]
We’re not currently accepting the Nexus-generated tokens; are you not using PATRIC tokens ?

On Jul 30, 2015, at 11:06 AM, nconrad <notifications@github.com<mailto:notifications@github.com>> wrote:


When I trying to use the python client with a username and password, I was getting the following from the workspace service:

api.patric.workspace.ServerError: JSONRPCError: -32603. Token validation failed: Failed to verify token: Token signed by unrecognized source: https://nexus.api.globusonline.org/goauth/keys/fb4f1e06-322c-11e5-9bf4-123139260d4e at /disks/p3/deployment/lib/Bio/KBase/AuthToken.pm line 481.

It looks like the bad method is https://github.com/PATRIC3/Workspace/blob/master/lib/biop3/Workspace/WorkspaceClient.py#L27

**Comment by nconrad** [comment]
Yeah, I understand now.  I'm using RAST and PATRIC tokens instead.   Just thought I'd point it out.
